GNUmakefile

```diff
@@ -60,11 +60,10 @@ test-compile:
 # 	go tool cover -html=c.out
 
 cibuild:
-	go build
-#	env GOOS=darwin GOARCH=amd64 go build
-#	env GOOS=windows GOARCH=amd64 go build
-#	env GOOS=linux GOARCH=amd64 go build
-#	env GOOS=linux GOARCH=ppc64 go build
+	go get github.com/mitchellh/gox
```

Comment:
this should be left to a step on the makefile to install all the dependencies, instead of installing it every time this is run